### PR TITLE
Audio module fully unlocked! 

### PR DESCRIPTION
**Pull request by Aask42**
Enjoy!

### CODE REVIEW

**Comment by Ancient123** [comment]
🎉 🎶🎵🎼

File: Software/dczia26_final/dczia26_final.ino [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
 
 // split into functional regions
-#include "dczia26.h"
+#include "Key.h" // This was referred to as dczia.h previously
 #include "dczia26_keypad.h"
 #include "dczia26_led.h"
 #include "dczia26_oled.h"
 #include "dczia26_ble.h"
 #include "dczia26_sd.h"
+#include "arduinoFFT.h" // Standard Arduino FFT library
```
Comment (rlucero) [inline Software/dczia26_final/dczia26_final.ino]: Just confirming, this is the `1..5.0` version of the library? It worked when I built it, but just want to book keep it for the README.

File: Software/dczia26_final/dczia26_oled.cpp [hunk 2]
```diff
@@ -216,5 +216,15 @@ void oled_displayCredits(Adafruit_SSD1306 *screen){
...
   screen->display();
+  delay(2000);
+  screen->clearDisplay();
+  screen->setTextSize(2);
+  screen->setTextColor(WHITE);
+  screen->setCursor(0, 0);
+  screen->println("Credits!");
+  screen->setTextSize(1);
+  screen->println("audio EQ by:");
+  screen->println("Aask");
```
Comment (rlucero) [inline Software/dczia26_final/dczia26_oled.cpp]: 👍

File: Software/dczia26_final/dczia26_final.ino (hunk 1 above)
Comment (Aask42) [reply Software/dczia26_final/dczia26_final.ino]: arduinoFFT 1.5.0 is indeed correct!

**Pull request merged**